### PR TITLE
[SD-1424] - Direct access to an item using its Unique Name

### PR DESCRIPTION
**Pull request by sjunaid**

### CODE REVIEW

File: app/scripts/superdesk-search/views/item-globalsearch.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <div class="popup advanced-globalsearch">
+        <div class="content dropdown">
+            <div class="advanced" ng-click="$event.stopPropagation()">
+            	<button class="close" tabindex="4" ng-click="close();"><i class="icon-close-small"></i></button>
+        		<div class="content"> 
+        			<form>
+                        <fieldset class="span-2">
+                            <div class="field">
+                               <label>Story Name</label>
+                               <input type="text" id="globalsearch-storyname" ng-keyup="openOnEnter($event)" ng-model="meta.unique_name" tabindex="1" sd-autofocus ng-if="flags.enabled">
```
Comment (eozan) [inline app/scripts/superdesk-search/views/item-globalsearch.html]: id is not needed anymore.

**Comment by eozan** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

**Pull request merged**